src/containers/shared/components/Transaction/XChainAddAttestation/Simple.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { TransactionSimpleComponent, TransactionSimpleProps } from '../types'
+import { XChainBridge } from '../XChainBridge'
+import { AccountCreateAttestation } from './AccountCreateAttestation'
+import { ClaimAttestation } from './ClaimAttestation'
+
+export const Simple: TransactionSimpleComponent = (
+  props: TransactionSimpleProps,
+) => {
+  const {
+    data: {
+      instructions: {
+        lockingDoor,
+        lockingIssue,
+        issuingDoor,
+        issuingIssue,
+        accountCreateAttestations,
+        claimAttestations,
+      },
+    },
+  } = props
+
+  return (
+    <>
+      <XChainBridge
+        lockingDoor={lockingDoor}
+        lockingIssue={lockingIssue}
+        issuingDoor={issuingDoor}
+        issuingIssue={issuingIssue}
+        bridgeOwner=""
```

Comment:
Why is `bridgeOwner` empty?  - Is that a field which could be updated later on?

Reply:
It's empty because of the TODO below.